servers/quarkus-server/src/test/java/org/projectnessie/server/TestBasicOperations.java

```diff
@@ -16,6 +16,9 @@
 package org.projectnessie.server;
 
 import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import org.projectnessie.server.profiles.QuarkusNativeProfileInmemory;
 
 @QuarkusTest
+@TestProfile(QuarkusNativeProfileInmemory.class)
```

Comment:
Do we need to apply this to _unit_ tests that run Quarkus in a JVM?

Reply:
argh - damn copy+paste bug. should be `QuarkiusProfileInmemory`

Reply:
good catch BTW!